modules/administration_manual/pages/maintenance/encryption/migrating-from-user-key-to-master-key.adoc

```diff
@@ -0,0 +1,102 @@
+= Migrating User Key Encryption to Master Key Encryption
+
+If you need to migrate from User Key-based to Master Key-based encryption, there are several steps that you need to follow to ensure a smooth and complete transition. 
+These are:
+
+. xref:disable-user-key-based-encryption[Disable User Key-based encryption]
+. xref:remove-the-encryption-records-from-the-owncloud-database[Remove the encryption records from the ownCloud database]
+. xref:remove-the-`files_encryption`-directory[Remove the `files_encryption` folder]
```

Comment:
1. for consistency: ``[Remove the `files_encryption` folder]`` --> ``[Remove the `files_encryption` directory]``
2. I do not think that this is a valid linke but a copy/paste error: ``xref:remove-the-`files_encryption`-directory``.
The correct link as it is also found on line 50 would imho be: ``xref:remove-the-files_encryption-directory`` (without the ticks)

Reply:
Well spotted. Thanks.

Reply:
Section updated.